index.js

```diff
@@ -17,18 +21,35 @@ module.exports = options => {
 	}, options);
 
 	const handleError = (title, err) => {
-		const showErrorBox = (electron.dialog || electron.remote.dialog).showErrorBox;
+		const isReady = (electron.app || electron.remote.app).isReady();
+		const dialog = (electron.dialog || electron.remote.dialog);
 
 		try {
 			options.logger(err);
 		} catch (err2) { // eslint-disable-line unicorn/catch-error-name
-			showErrorBox('The function specified in the `logger` option in electron-unhandled threw an error', err2.stack);
+			dialog.showErrorBox('The function specified in the `logger` option in electron-unhandled threw an error', err2.stack);
 			return;
 		}
 
 		if (options.showDialog) {
 			const stack = err ? (err.stack || err.message || err || '[No error message]') : '[Undefined error]';
-			showErrorBox(title, cleanStack(stack));
+
+			if (isReady) {
+				const btnIndex = dialog.showMessageBox({
+					type: 'error',
+					buttons: [process.platform === 'darwin' ? 'Copy Error' : 'Copy error', 'OK'],
+					defaultId: 1,
+					title,
+					noLink: true,
+					message: err.message,
```

Comment:
Like this ?

`const message = err ? (err.message || err.stack || err || '[No error message]') : '[Undefined error]';`